02-electric/electric-section-one/electric-section-one.css

```diff
@@ -17,71 +31,296 @@
 
 .electric-section-one-title {
   text-align: center;
-  font-size: var(--title-size-mobile);
+  font-size: var(--title-font-size-mobile);
+  font-weight: var(--title-font-weight-mobile);
 }
 
-.electric-section-one-subtitle {
-  text-align: center;
-  font-size: var(--title-size-mobile);
+@media (min-width: 500px) {
+  .electric-section-one-title {
+    text-align: center;
+    font-size: var(--title-font-size-desktop);
+    font-weight: var(--title-font-weight-desktop);
+  }
 }
 
-
 /*****PART ONE******/
 
-.electric-section-one-part-one-img {
+.electric-section-one-part-one-container{
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-areas:
+    "electric-section-one-part-one-image-container"
+    "electric-section-one-part-one-copy-container";
+  row-gap: var(--subcontainer-row-gap-mobile);
+  column-gap: none;
+  /* ^for the media query after */
+  border: var(--subcontainer-border-mobile);
+  background-color: var(--level2-background-color);
+}
+
+@media (min-width:500px) {
+  .electric-section-one-part-one-container{
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-areas:
+    "electric-section-one-part-one-image-container electric-section-one-part-one-copy-container";
+    column-gap: var(--subcontainer-column-gap-desktop);
+    padding: var(--subcontainer-padding-desktop);
+  }
+}
+
+.electric-section-one-part-one-image-container{
+  grid-area: electric-section-one-part-one-image-container;
+  display: grid;
+  justify-content: center;
+  align-content: center;
+  background-color: var(--level3-background-color);
+}
+
+@media (min-width: 500px) {
+  .electric-section-one-part-one-image-container {
+  }
+}
+
+.electric-section-one-part-one-image {
   display: block;
   width: 100%;
 }
 
-.electric-section-one-part-one-title {
+@media (min-width: 500px) {
+  .electric-section-one-part-one-image {
+  }
+}
+
+.electric-section-one-part-one-copy-container{
+  grid-area: electric-section-one-part-one-copy-container;
+  display: grid;
+  align-content: center;
+  row-gap: var(--copy-container-row-gap-mobile);
+  background-color: var(--level3-background-color);
+}
+
+.electric-section-one-part-one-copy-container >*{
```

Comment:
as above, remove all of these and define them at the top

Reply:
done